tests/tablespaces/test_tablespaces_03.py

```diff
@@ -0,0 +1,16 @@
+from nose.tools import *
+from tests.tablespaces.tablespace_utils import node1
+from tests.tablespaces.tablespace_utils import node2
+
+
```

Comment:
That's where I expected to see the test_001 launching the failover as in the other (non-tablespace) tests, if that's possible. If we keep the approach where a docker-compose command is used in the Makefile, I suppose a comment here would make sense too.